configure.ac

```diff
@@ -2411,6 +2411,67 @@ then
 fi
 # }}}
 
+# --with-libdpdk {{{
+AC_ARG_WITH(libdpdk, [AS_HELP_STRING([--with-libdpdk@<:@=PREFIX@:>@], [Path to the DPDK build directory.])],
+[
+	if test "x$withval" != "xno" && test "x$withval" != "xyes"
+	then
+		with_dpdk_path="$withval"
+		with_libdpdk="yes"
+	else
+		if test "x$withval" = "xno"
+		then
+			with_libdpdk="no (disabled)"
+		fi
+	fi
+], [with_libdpdk="no"])
+
+if test "x$with_libdpdk" = "xyes"
+then
+	RTE_BUILD="$with_dpdk_path"
+	DPDK_INCLUDE="$RTE_BUILD/include"
+	AC_CHECK_FILE([$DPDK_INCLUDE/rte_config.h], [LOCAL_DPDK_INSTALL=true],
```

Comment:
true will work since shell interprets it as a string, but shell doesn't have boolean values. Please just use "yes" here.